src/libLookup/Lookup.cpp

```diff
@@ -722,7 +717,14 @@ bool Lookup::SetDSCommitteInfo() {
       struct in_addr ip_addr;
       inet_pton(AF_INET, v.second.get<string>("ip").c_str(), &ip_addr);
       Peer peer((uint128_t)ip_addr.s_addr, v.second.get<unsigned int>("port"));
-      m_mediator.m_DSCommittee->emplace_back(make_pair(key, peer));
+
+      if (replaceMyPeerWithDefault && (key == m_mediator.m_selfKey.second)) {
```

Comment:
Can put replaceMyPeerWithDefault as second condition instead of first?

Reply:
Since the first condition is faster to execute (just a bool check), I think it's better to keep it first.  We don't want to unnecessarily do a key comparison every time first.